src/bin/release-please.ts

```diff
@@ -89,6 +89,21 @@ const argv = yargs
         .option('label', {
           default: 'autorelease: pending',
           describe: 'label to remove from release PR',
+        })
+        .option('release-type', {
+          describe: 'what type of repo is a release being created for?',
+          choices: [
+            'node',
```

Comment:
I'm guessing this list should now include `simple`?

Reply:
I think we missed this in the earlier PR yes, I'll correct.